### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chauthai**
Fixed wget command to add `rocm.gpg.key` to sources.list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -242,11 +242,9 @@ To install from a Debian Repository:
 
      For Debian-based systems like Ubuntu, configure the Debian ROCm repository as follows:
    
-      	wget -q0 – http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | 
-
-      	sudo apt-key add -echo 'deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main' | 
-
-      	sudo tee /etc/apt/sources.list.d/rocm.list
+      	wget -qO - http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key |
+        sudo sudo apt-key add -sudo sh -c 'echo deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main
```
Comment (icarus-sparry) [inline README.md]: Whilst the original code is clearly wrong, this PR is not the fix. The "sudo sudo" is a major red flag.
There should be a newline between the "-" and the "echo" in the "-echo" on line 247. So the code should read

      wget -q0 – http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | 
           sudo apt-key add -

      echo 'deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main' |
           sudo tee /etc/apt/sources.list.d/rocm.list

File: README.md [hunk 2]
```diff
@@ -242,11 +242,9 @@ To install from a Debian Repository:
...
      For Debian-based systems like Ubuntu, configure the Debian ROCm repository as follows:
    
-      	wget -q0 – http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | 
-
-      	sudo apt-key add -echo 'deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main' | 
-
-      	sudo tee /etc/apt/sources.list.d/rocm.list
+      	wget -qO - http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key |
+        sudo sudo apt-key add -sudo sh -c 'echo deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main 
+        > /etc/apt/sources.list.d/rocm.list'
```
Comment (t-tye) [inline README.md]: These lines should be replaced by the following 2 lines, with a line containing ``` before and after them to ensure the lines are treated as a code block:

```
wget -qO - http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | sudo apt-key add -
sudo sh -c 'echo deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main > /etc/apt/sources.list.d/rocm.list'
```

File: README.md [hunk 3]
```diff
@@ -242,11 +242,9 @@ To install from a Debian Repository:
...
      For Debian-based systems like Ubuntu, configure the Debian ROCm repository as follows:
    
-      	wget -q0 – http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | 
-
-      	sudo apt-key add -echo 'deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main' | 
-
-      	sudo tee /etc/apt/sources.list.d/rocm.list
+        wget -qO - http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | sudo apt-key add -
+        
+        echo 'deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main' | sudo tee /etc/apt/sources.list.d/rocm.list
```
Comment (t-tye) [inline README.md]: Remove blank line and surround by ``` so that formatting will preserve this as a code block.

**Comment by icarus-sparry** [comment]
@chauthai My apologies, I missed your update fixing the typos. Thank you for the fix.

**Pull request merged**